MathMagicianTests/Numbers/FibonacciNumberTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MathMagician.Numbers;
+
+namespace MathMagicianTests.Numbers
+{
+    class FibonacciNumberTests
```

Comment:
@jessicakilby This test class is missing its [`TestClass` annotation](https://github.com/nss-evening-cohort-04/mathemagician/pull/24/files#diff-6992dfd56e65d59dce71597eb99efeb2R7). Without it, Visual Studio test runner won't find the tests.